### ISSUE TITLE
Not compatible with Vue3

### ISSUE THREAD

**Issue by Shion1305** [paupaulaz/VueJSTwitterFeed#1]
I'm not sure, but maybe his component did not work with Vue3.

**Comment by maietta** [comment]
You are correct, it does not work in Vue 3. Disappointed that there isn't a lot of interest in working on a Vue 3 version I will attempt it at some point in the future if it hasn't been worked on by then as I need something working for a project I am working on. I've never worked on a vue plugin that can be installed via NPM, so it will take me some time to get that setup.

I will make sure to label it something like vue3-twitter-timeline so that it's clear that it's a Vue 3 version.
